reviewdog.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/reviewdog/reviewdog/difffilter"
 )
 
+var X = 14
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported var X should have comment or be unexported